app/assets/stylesheets/application.scss

```diff
@@ -125,3 +127,18 @@ a:hover {
 .panel #created {
   color: $font-color;
 }
+
+.field label {
+  color: $font-color;
+  font-size: 1em;
+}
+
+.tag {
+  border-style: solid;
+  border-width: 1px;
+  border-color: #d8d8d8;
+  border-radius: 5px;
+  padding: .15rem .40rem;
```

Comment:
`.15` should be written with a leading zero as `0.15`<br>`.40` should be written with a leading zero as `0.40`